librz/core/canalysis.c

```diff
@@ -5755,6 +5755,105 @@ RZ_IPI bool rz_core_analysis_var_rename(RzCore *core, const char *name, const ch
 	return true;
 }
 
+static bool is_in_data_map(RzCore *core, const ut64 address) {
+	if (address < 256 || address == UT64_MAX) {
```

Comment:
Why? It looks somewhat arbitrary. At least make it a define or something?

Reply:
No, this is not useless. there are a lot of false positives when it comes to addresses lower than 256, just imagine for example:
```
@@ --1,14 +-1,14 @@
 |       ,=< 0x000113bd      je    0x1144e
-|       |   0x000113c3      mov   esi, 0x2f                            ; '/' ; int c
+|       |   0x000113c3      mov   esi, 0x2f                            ; data.0000002f ; int c
 |       |   0x000113c8      call  sym.imp.strrchr                      ; char *strrchr(const char *s, int c)
 |       |   0x000113cd      test  rax, rax
 |      ,==< 0x000113d0      je    0x11424
 |      ||   0x000113d2      lea   rdx, [rax + 1]
 |      ||   0x000113d6      mov   rcx, rdx
 |      ||   0x000113d9      sub   rcx, rbx
-|      ||   0x000113dc      cmp   rcx, 6
+|      ||   0x000113dc      cmp   rcx, 6                               ; data.00000006
 |     ,===< 0x000113e0      jle   0x11424
 |     |||   0x000113e2      lea   rsi, [rax - 6]
 |     |||   0x000113e6      mov   ecx, 7
```

Reply:
Then please add a comment.

Reply:
i did.